astropy/visualization/wcsaxes/core.py

```diff
@@ -572,7 +585,7 @@ def _get_transform_no_transdata(self, frame):
                 else:
                     return pixel2world + CoordinateTransform(self.wcs, frame)
 
-    def get_tightbbox(self, renderer):
+    def get_tightbbox(self, renderer, *args, **kwargs):
```

Comment:
Shouldn't something be done with args and kwargs?

Reply:
In principle yes but I'm deferring that to a future PR. The args and kwargs are matplotlib-version-dependent.

Reply:
Could you add a note to the code to state that it is for the future?